examples/ReorderExample.js

```diff
@@ -47,15 +45,18 @@ class ReorderExample extends React.Component {
         'street',
         'zipCode',
         'sentence',
-        'companyName'
+        'companyName',
       ],
+      recycling: {},
     };
 
-    this._onColumnReorderEndCallback = this._onColumnReorderEndCallback.bind(this);
+    this._onColumnReorderEndCallback = this._onColumnReorderEndCallback.bind(
+      this
+    );
+    this.toggleCellsRecycling = this.toggleCellsRecycling.bind(this);
```

Comment:
turn these into lambda functions to avoid binding here

Reply:
Done